src/SmartComponents/Inventory/applications/advisor/Advisor.js

```diff
@@ -79,84 +94,150 @@ class InventoryRuleList extends Component {
-                            <CardHeader>
-                                <CommentSlashIcon size='lg'/>
-                            </CardHeader>
-                            <CardBody>
-                                No issues reported by Insights for this system at the moment.
-                            </CardBody>
-                        </Card>
-                ) }
-                { inventoryReportFetchStatus === 'failed' && this.props.entity && (
-                    <Card className="ins-empty-rule-cards">
-                        <CardHeader>
-                            <FrownOpenIcon size='lg'/>
-                        </CardHeader>
-                        <CardBody>
-                            There was an error fetching rules list for this Entity. Please show your administrator this screen.
-                        </CardBody>
-                    </Card>
-                ) }   { inventoryReportFetchStatus === 'failed' && !this.props.entity && (
-                    <Card className="ins-empty-rule-cards">
-                        <CardHeader>
-                            <FrownOpenIcon size='lg'/>
-                        </CardHeader>
-                        <CardBody>
-                            This system can not be found or might no longer be registered to Red Hat Insights.
-                        </CardBody>
-                    </Card>
-                ) }
-            </Fragment>
-        );
+        const { remediation, inventoryReportFetchStatus, rows } = this.state;
+        const results = rows ? rows.length / 2 : 0;
+        return <Fragment>
+            <TableToolbar className='pf-u-justify-content-space-between'>
+                <ToolbarGroup>
+                    <ToolbarItem>
+                        <RemediationButton
+                            isDisabled={ !remediation }
+                            dataProvider={ () => remediation }
+                            onRemediationCreated={ result => this.props.addNotification(result.getNotification()) }/>
+                    </ToolbarItem>
+                    <ToolbarItem>{ this.buildKebab() }</ToolbarItem>
+                </ToolbarGroup>
+                <ToolbarGroup>
+                    <ToolbarItem>
+                        { results === 1 ? `${results} rule` : `${results} rules` }
+                    </ToolbarItem>
+                </ToolbarGroup>
+            </TableToolbar>
+            { inventoryReportFetchStatus === 'pending' && (
+                <Card>
+                    <CardBody>
+                        <List/>
+                    </CardBody>
+                </Card>
+            ) }
+            { inventoryReportFetchStatus === 'fulfilled' && (results > 0 ?
+                <Table aria-label={ 'rule-table' } onCollapse={ this.handleOnCollapse } rows={ rows } cells={ [ '' ] }>
+                    <TableHeader/>
+                    <TableBody/>
+                </Table>
+                :
+                <Card>
```

Comment:
Is it correct in Card? Other empty states are outside card component.

Reply:
We need to have the white background and margin around the the message state, i didn't build it into messagestate.js (cuz maybe we wanta abstract it somewhere else n drop it somewhere else 🤷‍♂) by default empty states don't have a background color

Reply:
Was this in the UX mocks? Other places aren't using backgrounds on their empty states. For consistency, this should be revisited.

Reply:
@katierik can you confirm or deny, white background for empty states?

Reply:
Yes white background. [Mock](https://redhat.invisionapp.com/share/V4NY3C6XKRW#/356911950_Empty_State_-_No_Insights). 
The difference between a grey bg and a white bg is this:
- Empty state should be directly on the grey bg if there is a configuration reason it is empty (i.e. it isn't connected yet)
- Empty state should be on the white bg if it is properly configured, but there just isn't anything to show! (e.g. no results after filtering; the system/rule happens to have 0 hits)